repos/system_upgrade/el7toel8/actors/tcpwrappersconfigread/libraries/library.py

```diff
@@ -0,0 +1,57 @@
+import errno
+import re
+
+from leapp.libraries.stdlib import api
+from leapp.models import DaemonList, TcpWrappersFacts
+
+CONFIG_FILES = ['/etc/hosts.allow', '/etc/hosts.deny']
+
+
+def read_file(path):
+    with open(path, 'r') as f:
+        return f.read()
+
+
+def get_lines(content):
+    content = re.sub(r'\\\n', '', content)
+    return content.split('\n')
+
+
+def is_comment(line):
+    return len(line) == 0 or line.isspace() or line.startswith('#')
+
+
+def get_daemon_list_in_line(line):
+    daemon_list = line.split(':', 1)[0]
+    daemon_list = daemon_list.replace(',', ' ')
+    daemon_list = daemon_list.replace('\t', ' ')
+    daemon_list = daemon_list.split(' ')
```

Comment:
nit: wondering if re.split(',| |\t', daemon_list) is more readable

Reply:
Nice, thanks! I didn't realize that exists :).

Reply:
Done. I've also extended the test coverage a bit.